### PR TITLE
Adding details about new recipes:update command

### PR DESCRIPTION
**Pull request by weaverryan**
See: https://github.com/symfony/flex/pull/845

The `versionadded` version may still change if we decide to put this into the 1.x branch.

### CODE REVIEW

File: setup/_update_recipes.rst.inc [hunk 1]
```diff
@@ -9,30 +9,25 @@ it's a good idea to keep your files in sync with the recipes.
...
 
-    The recipes commands were introduced in Symfony Flex 1.6.
+    The ``recipes:update`` command was introduced in Symfony Flex 2.1.
 
 .. code-block:: terminal
 
+    # choose an outdated recipe to update
+    $ composer recipes:update
+
+    # update a specific recipes
```
Comment (gnito-org) [inline setup/_update_recipes.rst.inc]: "update a specific recipes" should be singlular.

Comment (weaverryan) [reply setup/_update_recipes.rst.inc]: Thanks :)

**Review by javiereguiluz — APPROVED** [review]
Fantastic feature 😍  Thanks for implementing it!

**Comment by wouterj** [comment]
Thanks @weaverryan!

As this is about Symfony Flex, I've merged it into 4.4 instead of 5.4.

**Pull request merged**